### PR TITLE
add note about CSP

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4529

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/federated-authentication-device-enrollment.md [hunk 1]
```diff
@@ -167,6 +167,9 @@ AuthenticationServiceUrl?appru=<appid>&amp;login_hint=<User Principal Name>
 
 After authentication is complete, the auth server should return an HTML form document with a POST method action of appid identified in the query string parameter.
 
+> [!NOTE]
+> To make an application compatible with strict CSP (Content Security Policy) it is usually necessary to make some changes to HTML templates and client-side code, add the policy header, and test that everything works properly once the policy is deployed.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/federated-authentication-device-enrollment.md]:
```suggestion
> To make an application compatible with strict CSP (Content Security Policy), it is usually necessary to make some changes to HTML templates and client-side code, add the policy header, and test that everything works properly once the policy is deployed.
```

**Comment by mypil** [comment]
@VLG17 - please merge the commit on the copy/edit review made by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@ManikaDhiman - PR has been copy edited and is ready for final review. This is based on issue #4529. 

Thank you.

**Comment by mypil** [comment]
@ManikaDhiman - Please assist to merge the PR content. Thank you.

**Pull request merged**